### ISSUE TITLE
Setup Wizard: add Talkback option in the UI

### ISSUE THREAD

**Issue by oppressor1761** [GrapheneOS/os-issue-tracker#8215]
Feature Request
Currently gos Setup Wizard does not have a UI to adjust Talkback options. Users can activate Talkback using Accessibility shortcuts though during setup wizard. 
I suggest to add Talkback option into Setup Wizard. As contrast, stock OS has the Talkback option in the Setup Wizard. 
Gos:

<img width="1279" height="1706" alt="Image" src="https://github.com/user-attachments/assets/e31f191a-686f-492a-9526-13700108550e" />

Stock:

<img width="1279" height="1706" alt="Image" src="https://github.com/user-attachments/assets/f1c75f62-1c93-4325-9ca9-3a6f4f1fa255" />

**Comment by oppressor1761** [comment]
Moved to https://github.com/GrapheneOS/platform_packages_apps_SetupWizard2/issues/50
